pubtools/pulplib/_impl/criteria.py

```diff
@@ -219,6 +219,28 @@ def in_(cls, values):
         """
         return InMatcher(values)
 
+    @classmethod
+    def less_than(cls, value):
+        """
+        Returns a matcher for a field whose value is less than the specified input
+        value
+
+        Arguments:
+            values (object)
+                An object to match against the field
+
+        Example:
+            .. code-block:: python
+
+            # would match where last_publish is before "2019-08-27T00:00:00Z
+            # date is reqired to be in this format
```

Comment:
```suggestion
            # date is required to be in this format
```

But anyway, I'm probably not agreeing with the example (commented below).